cli/cmd/project/connect.go

```diff
@@ -154,6 +154,7 @@ func ConnectCmd(cfg *config.Config) *cobra.Command {
 	connectCmd.Flags().StringVar(&name, "name", "", "Project name (default: the Github repo name)")
 	connectCmd.Flags().StringVar(&description, "description", "", "Project description")
 	connectCmd.Flags().BoolVar(&public, "public", false, "Make dashboards publicly accessible")
+	connectCmd.Flags().StringVar(&cfg.DefaultOrg, "org", cfg.Org(), "Organization Name")
```

Comment:
1. Make this a persistent flag on the `rill project` command (so it applies to all subcommands), instead of repeating in each command
2. Rename `cfg.DefaultOrg` to `cfg.Org`
3. Remove the `cfg.Org()` function, we can just access it directly